### PR TITLE
Hacktoberfest: trim the adminAddress before saving

### PR DESCRIPTION
**Pull request by darinpope**
Signed-off-by: Darin Pope <darin@planetpope.com>

### CODE REVIEW

**Review by MarkEWaite — APPROVED** [review]

**Review by sladyn98 — APPROVED** [review]

**Review by res0nance — APPROVED** [review]

**Review by MRamonLeon — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Review by daniel-beck — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
Thanks for your contribution! If this is a Hacktoberfest pull request, please let us know so that we can label it properly and facilitate reviews. Check out [this page](https://www.jenkins.io/events/hacktoberfest/) for more information about Jenkins in Hacktoberfest: guidelines, featured projects, etc. 

[image](https://www.jenkins.io/events/hacktoberfest/)

**Review by Wadeck — APPROVED** [review]
👍

**Review by markjacksonfishing — APPROVED** [review]

**Pull request merged**
